gato/attack/attack.py

```diff
@@ -32,13 +32,15 @@ def __init__(
         http_proxy: str = None,
         author_email: str = None,
         author_name: str = None,
-        timeout: int = 30
+        timeout: int = 30,
+        github_url: str = None,
```

Comment:
Since there isn't a scenario where the github url passed to Api would ever be None, we should make this a required variable.